Framework/Kernel/test/ArrayPropertyTest.h

```diff
@@ -301,6 +301,65 @@ class ArrayPropertyTest : public CxxTest::TestSuite {
                       static_cast<Property *>(0))
   }
 
+  void testPrettyPrinting() {
+    std::vector<std::string> inputList{
+        "1,2,3", "-1,0,1", "356,366,367,368,370,371,372,375", "7,6,5,6,7,8,10",
+        "1-9998, 9999, 2000, 20002-29999"};
+    std::vector<std::string> resultList{"1-3", "-1-1",
```

Comment:
Does it make sense to make these vectors `const`? Similarly to other input and result lists below.